contrib/debian/control

```diff
@@ -0,0 +1,23 @@
+Source: netdata
+Build-Depends: debhelper (>= 9),
+ dpkg-dev (>= 1.13.19),
+ dh-systemd (>= 1.5),
+ zlib1g-dev,
+ autoconf,
+ automake,
+ pkg-config
+Section: net
+Priority: optional
+Maintainer: Costa Tsaousis <costa@tsaousis.gr>
```

Comment:
This should be the debian maintainer name. Upstream author is in debian/copyright